parser-typechecker/src/Unison/Codebase/Serialization/V1.hs

```diff
@@ -479,7 +482,8 @@ getPattern getA = getWord8 >>= \tag -> case tag of
       <*> getPattern getA
       <*> getSeqOp
       <*> getPattern getA
-  12 -> Pattern.TextP <$> getA <*> getText 
+  12 -> Pattern.TextP <$> getA <*> getText
+  13 -> Pattern.CharP <$> getA <*> getChar
```

Comment:
Good. I'm sure you figured this out, but this is the right move, any new constructors get the next largest available tag word, so that existing serialized ASTs still deserialize fine. We never reassign or reshuffle the tags unless we are prepared to do a codebase format upgrade, which we will try to do as infrequently as possible.